app/controllers/user_challenges_controller.rb

```diff
@@ -27,6 +28,28 @@ def destroy
     end
   end
 
+  def evaluate_comments
+    evaluator = UserChallenge::Evaluator.new(params: params)
+    result = evaluator.save_comments
+
+    if result.success?
+      head :no_content
+    else
+      render json: { resource: 'user_challenge', errors: [user_challenge: result.errors.first] }, status: 500
```

Comment:
Drive by comment, a [`500`](https://httpstatuses.com/500) status is probably what you want to go for here.

500's should only be raised when something like an exception is raised or something on the server is really borked.

I would look at something in the `4xx` range.